src/Integration.Vsix/Integration.Vsix.csproj

```diff
@@ -452,6 +452,7 @@
 
   <Target Name="IncludeNuGetPackageReferences" AfterTargets="GetVsixSourceItems">
     <ItemGroup>
+      <VSIXSourceItem Include="@(ReferenceCopyLocalPaths)" Condition="'%(ReferenceCopyLocalPaths.NuGetPackageId)' == 'NewtonSoft.Json'" />
```

Comment:
Added this line so the dll gets included in the VSIX. I checked both 2019 and 2022 VSIXes to make sure it is there.